Datadog/Datadog.xcodeproj/xcshareddata/xcschemes/DatadogCore iOS.xcscheme

```diff
@@ -196,6 +196,26 @@
                BlueprintName = "DatadogCoreTests iOS"
                ReferencedContainer = "container:Datadog.xcodeproj">
             </BuildableReference>
+            <SkippedTests>
+               <Test
+                  Identifier = "HeadBasedSamplingTests/testSamplingLocalTrace()">
+               </Test>
```

Comment:
💡 These tests are disabled on purpose. I propose we go red → green through `RUM-3470` (next PR) and later `RUM-3535` ("trace context injection control").